docs/resources/integration_email.md

```diff
@@ -0,0 +1,73 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_integration_email Resource - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Send an email message to your ticket system (or any destination).
+---
+
+# mondoo_integration_email (Resource)
+
+Send an email message to your ticket system (or any destination).
+
+## Example Usage
+
+```terraform
+provider "mondoo" {
+  space = "hungry-poet-123456"
+}
+
+# Setup the Email integration
+resource "mondoo_integration_email" "email_integration" {
+  name = "My Email Integration"
+
+  recipients = [
+    {
+      name          = "John Doe"
+      email         = "john@example.com"
+      is_default    = true
+      reference_url = "https://example.com"
+    },
+    {
+      name          = "Alice Doe"
+      email         = "alice@example.com"
+      is_default    = false
+      reference_url = "https://example.com"
+    }
+  ]
```

Comment:
Instead of an array, could we do objects?
```suggestion
    recipient {
      name          = "John Doe"
      email         = "john@example.com"
      is_default    = true
      reference_url = "https://example.com"
    }
    
    recipient {
      name          = "Alice Doe"
      email         = "alice@example.com"
      is_default    = false
      reference_url = "https://example.com"
    }
```

Reply:
@afiune I do not recommend to use individual blocks for each recipient instead of a list. As a user I would feel it to be more intuitive if I could define it in one single list instead of having to define multiple bocks.

Furthermore Bocks like shown in this image are only supposed to be used for legacy projects:
<img width="746" alt="Untitled" src="https://github.com/user-attachments/assets/973dce6b-8210-4eee-bb75-ccab57217611">

Reply:
Great observation. Let's not implement any legacy code.